.run/DESIGNER No AUTH.run.xml

```diff
@@ -23,9 +23,9 @@
       <env name="FEEDBACK_LINK" value="http://localhost:3008/feedback" />
       <env name="JWT_AUTH_COOKIE_NAME" value="fsd_user_token" />
       <env name="JWT_AUTH_ENABLED" value="false" />
-      <env name="JWT_REDIRECT_TO_AUTHENTICATION_URL" value="http://localhost:3004/sessions/sign-out" />
+      <env name="JWT_REDIRECT_TO_AUTHENTICATION_URL" value="http://localhost:4004/sessions/sign-out" />
```

Comment:
Can we fix the domains on these all as well?

Reply:
sure @samuelhwilliams

Reply:
@nuwan-samarasinghe @albertkol i've added the full localhost domains for everything - can one of you check that it works correctly, and if so approve and we can merge this in?

Reply:
I have no idea what the xml files do.. :shrug: